create-proxy.sh

```diff
@@ -30,11 +33,17 @@ fi
 echo "Creating container [${NAME}] using image [${CONTAINER_IMAGE}:${CONTAINER_VERSION}]."
 
 if [ -e "${PSK_FILE}" ]; then
-  COMMAND="chown zabbix:zabbix \"/var/lib/${PSK_FILE}\"; chmod 600 \"/var/lib/${PSK_FILE}\"; ${START_CMD}"
-else
-  COMMAND="${START_CMD}"
+  COMMAND="chown zabbix:zabbix \"/var/lib/${PSK_FILE}\"; chmod 600 \"/var/lib/${PSK_FILE}\";"
+if [ -e "${CA_FILE}" ]; then
```

Comment:
Probably command should change ownership of all ca, key and cert files if they are given. Now this logic only overrides COMMAND variable so that only the latest chown (for cert file) will be made.